docs/organization-integration/onpremise-integration.md

```diff
@@ -156,14 +161,14 @@ stages:
 
 Next, you will need to schedule *when* you're going to publish your system to Sigrid. Azure DevOps allows you to [schedule pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml). You can use this to periodically publish your system to Sigrid, for example as a nightly build. Alternatively, you can use Sigrid CI on a more continuous basis by running the pipeline step after every commit.
 
-## Functional differences in Sigrid on-premise
+## Functional/technical differences in Sigrid on-premise
 
 - The on-premise Sigrid distribution is single tenant. You cannot create your own "tenants", all systems and analyses will end up in your portfolio. That said, you can still use Sigrid’s user management to define which people should have access to which systems.
-- [Mendix Quality and Security Management](https://www.softwareimprovementgroup.com/mendix-quality-and-security-management-powered-by-sigrid/) is only available if you are using Mendix in combination with GitHub or GitLab.
-
-## Technical differences in Sigrid on-premise
-
+- You are required to use Sigrid CI for publishing source code. SFTP uploads and manual uploads are not supported.
```

Comment:
I'd phrase this slightly differently as while it needs to run from a CI pipeline, it's currently not using the `sigridci` script to publish the code for analysis and import. The reporting stage at the end however is the same.

Reply:
Fair point. How about this?